README.md

```diff
@@ -1,21 +1,66 @@
 # sdk.coverage.tests
 A repo to sync all sdks to the same tests
 
-# Adding new generic tests
+## Project structure
 
-## JS SDK's (@applitools/sdk-coverage-tests v2)
+1. This repo should contain all JS code related to generating tests. But not to running them. SDK-specific workflow scripts, or related code (e.g. base class or utility functions) in the specific language should exists in the relevant SDK repo.
 
-Create new test in `coverage-tests.js` inside this repo.
+2. Tests are specified in the file `coverage-tests.js`.
 
-## Non-JS SDK's ((@applitools/sdk-coverage-tests v1)
+3. There is a folder for each language which should contain the emitter, template, and overrides for that language. Example: https://github.com/applitools/sdk.coverage.tests/tree/master/js
 
-1. Create new test in `test.js` inside this repo.
-2. Updated `supported-test.js` file with the new tests inside SDK implementation of the generic tests
+4. In the specific SDK repo, there should be a `coverage-tests` folder with `package.json` that has the dependency of the cli, `@applitools/sdk-coverage-tests`, and a configuration for specifying where to take the code emitting files from. See example here: `//TODO add Java or C# folder url`
```

Comment:
What is the cli?

Reply:
The package `@applitools/sdk-coverage-tests` is referred to as "the cli". This is because it has an executable called `coverage-tests` which allows you to run commands in the, well, cli. You can see examples of this cli in the new readme: https://github.com/applitools/sdk.coverage.tests/pull/42/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R31

Reply:
@amitzur I thought so, but the commas confused me. Perhaps a different phrasing, like:

>  ... the dependency of `@applitools/sdk-coverage-tests` (a.k.a _the CLI_), and a configuration...

Reply:
Ah, i can understand the ambiguity now. I will change it. Thanks